duffy/cli.py

```diff
@@ -3,14 +3,18 @@
 import click
 import uvicorn
 
-from ..configuration import config, read_configuration
-from ..version import __version__
+from . import database
```

Comment:
```suggestion
from duffy.database.setup import setup_db_schema
from duffy.database import init_model
```

Reply:
I want to stick to relative imports within the package (`migrations/env.py` is an exception by way of how it's invoked by alembic, maybe this needs to be commented), and prefixing it with `database` is intentional, it saves me from typing a comment down below (about what model this call is about). I need to think about how best to do it, though.

Reply:
The changes that I listed in this thread of subcomments are "not" suggestions but rather, the things that I did to make the code work and continue reviewing the pull request. :)

Reply:
👍